scripts/php.sh

```diff
@@ -56,6 +60,7 @@ function run() {
 	cp -f /usr/share/doc/$VERSION-*/php.ini-development /etc/php.ini
 	sed -i "s/;date\.timezone.*/date\.timezone = ${TZ}/g" /etc/php.ini
 	sed -i "s/memory_limit.*/memory_limit = ${MEMORY_LIMIT}M/g" /etc/php.ini
+	sed -i "s/max_execution_time.*/max_execution_time = ${EXEC_TIME}/g" /etc/php.ini
```

Comment:
What may be better is to only update the value if it's currently less than the value we're looking to move to? But then a dev can't reduce the time via this script (which is fine, that's not it's purpose, I suppose - it's just for initial set-up of PHP)